core/src/test/resources/util/test-application-multi.yml

```diff
@@ -0,0 +1,29 @@
+spring:
+  profiles:
+    active: dev,qa
+
+---
+
+spring:
+  profiles: dev
```

Comment:
@rhuss - i dont think spring.profiles can have multiple values https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-profiles.html , so we dont need to worry on it . Only spring.profiles.active can have multiple values via comma seperation